### DISCUSSION TITLE
Replace brewed python

### DISCUSSION THREAD

**Discussion by pasteley** [Homebrew/discussions#3771] [Everyday usage]
Greetings, folks

Looking for an option to specify path for allready installed (via asdf or pyenv) python shims (and side-packages).
It looks like waste of space for me to keep another installation of the same runtime for 2-3 packages.

Is it possible somehow?

I can see `#!/opt/homebrew/opt/python@3.10/bin/python3` in shebang of binaries (in `/opt/homebrew/bin`), so it shouldn't be too hard to replace those along side with lib location, right?

**Comment by SMillerDev** [comment]
There is no such option, brew works well because we test our packages extensively and this would undermine that testing.

**Reply by pasteley** [reply]
Can you, please, explain how this could undermine testing?

**Reply by SMillerDev** [reply]
We don't test with your python install, we test with the brew dependencies

**Comment by carlocab** [comment]
I suggest just ignoring the fact that you have multiple Python installations on your machine, since there are likely to be others that you don't know about. If you have Xcode or the CLT installed, then you'll also have another Python3 install floating around (or two, if you have both of them installed).

Additionally, you might find that some apps you've installed into `/Applications` also ship their own copy of `python3`. I tried this command and found three of them:
```
find /Applications -name 'python*' -executable -print
```
